src/coreaudio.c

```diff
@@ -139,12 +139,15 @@ static void destroy_ca(struct SoundIoPrivate *si) {
     struct SoundIoCoreAudio *sica = &si->backend_data.coreaudio;
 
     AudioObjectPropertyAddress prop_address = {
-        kAudioHardwarePropertyDevices,
+        kAudioHardwarePropertyDefaultOutputDevice,
```

Comment:
I think we shouldn't replace `kAudioHardwarePropertyDevices`, because that will stop us from rescanning if an audio device is added which does not change the default input or output device.

Just adding listeners for `kAudioHardwarePropertyDefaultOutputDevice` and `kAudioHardwarePropertyDefaultInputDevice` should work, right?

Reply:
You are right. Changed my PR.